<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <meta name="generator" content="pandoc" />
    <meta name="viewport" content="width=device-width, initial-scale=1">

    <meta name="author" content="Jason Winget" />
    <meta name="date" content="2019-08-01" />
    <title>Jason Winget’s CV</title>

        <link href="winget_cv_files/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
    <link href="winget_cv_files/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />
    <link href="winget_cv_files/paged-0.3.1/css/resume.css" rel="stylesheet" />
    <script src="winget_cv_files/paged-0.3.1/js/config.js"></script>
    <script src="winget_cv_files/paged-0.3.1/js/paged.js"></script>
    <script src="winget_cv_files/paged-0.3.1/js/hooks.js"></script>
    
        <link rel="stylesheet" href="https:\\cdn.rawgit.com\jpswalsh\academicons\master\css\academicons.min.css" type="text/css" />
    
  </head>
  <body>

<div id="aside" class="section level1">
<h1>Aside</h1>
<p><img src="static/img/Jason.png" alt="Jason Winget" style="width:80.0%" /></p>
<div id="contact" class="section level2">
<h2>Contact Info</h2>
<ul>
<li><i class="fa fa-envelope"></i> <a href="mailto:jwinget@gmail.com" class="email">jwinget@gmail.com</a></li>
<li><i class="ai ai-google-scholar"></i> <a href="https://scholar.google.com/citations?user=uCmGx8YAAAAJ&amp;hl=en">Publications</a></li>
<li><i class="fa fa-github"></i> <a href="https://github.com/jwinget">github.com/jwinget</a></li>
<li><i class="fa fa-twitter"></i> <a href="https://twitter.com/iplusfour">@iplusfour</a></li>
<li>For more information, please contact me via email.</li>
</ul>
</div>
<div id="skills" class="section level2">
<h2>Skills</h2>
<ul>
<li><p>Proteomics (targeted and untargeted) from <em>in vitro</em> or <em>in vivo</em> samples</p></li>
<li><p>Additional experience in LC-MS/MS CCID of small organics</p></li>
<li><p>End-to-end scientist capable of design, execution, and data analysis/interpretation</p></li>
<li><p>Highly skilled in R. Also knowledgable of Python, HTML/CSS</p></li>
</ul>
</div>
<div id="disclaimer" class="section level2">
<h2>Disclaimer</h2>
<p>Last updated on 2019-08-01.</p>
</div>
</div>
<div id="main" class="section level1">
<h1>Main</h1>
<div id="title" class="section level2">
<h2>Jason Winget</h2>
<div id="senior-scientist-proteomics-lc-msms-and-data-science" class="section level3">
<h3>Senior Scientist, Proteomics, LC-MS/MS, and Data Science</h3>
<p>I am a PhD biochemist with 10 years of experience in proteomics, including
quantitative shotgun and targeted approaches. I am a strong advocate for reproducible
research, and have focused my personal development on best statistical practices for data
analysis and interpretation, as well as clear and transparent communication of results.</p>
<p>I am particularly interested in opportunities which leverage data science
(statistical analysis and modeling) to provide deeper insights that address biological questions
about mechanism of action and health.</p>
</div>
</div>
<div id="professional-experience" class="section level2" data-icon="suitcase">
<h2>Professional Experience</h2>
<div id="senior-scientist" class="section level3">
<h3>Senior Scientist</h3>
<p>Procter &amp; Gamble</p>
<p>Mason, OH, USA</p>
<p>Present - 2019</p>
<div class="concise">
<ul>
<li>Technical lead for corporate proteomics</li>
<li>Developed internal capability in Data-Independent Analysis (DIA)</li>
<li>Performed small organic chemical constitutent identification (CCID) to support current business needs</li>
<li>Supported naturals &amp; botanicals program</li>
</ul>
</div>
</div>
<div id="scientist" class="section level3">
<h3>Scientist</h3>
<p>Procter &amp; Gamble</p>
<p>Mason, OH, USA</p>
<p>2019 - 2016</p>
<div class="concise">
<ul>
<li>Developed high throughput targeted proteomics assays to support clinical studies.</li>
<li>Given CEO award for label-free proteomics approaches to refine safety assessments for Type-I allergy</li>
<li>Harmonized sample preparation, LC-MS/MS methods, and data analysis across studies</li>
<li>Taught “Introduction to R” courses</li>
</ul>
</div>
</div>
<div id="research-scientist" class="section level3">
<h3>Research Scientist</h3>
<p>Institute for Systems Biology</p>
<p>Seattle, WA, USA</p>
<p>2016 - 2014</p>
<div class="concise">
<ul>
<li>Performed shotgun and targeted proteomic analysis of human epidermis</li>
<li>Developed software for label-free quantification</li>
</ul>
</div>
</div>
<div id="postdoctoral-researcher" class="section level3">
<h3>Postdoctoral Researcher</h3>
<p>Institute for Systems Biology</p>
<p>Seattle, WA, USA</p>
<p>2014 - 2012</p>
<div class="concise">
<ul>
<li>Method development for proteomic analysis of low protein yield clinical skin surface samples</li>
<li>Refined data processing parameters for samples containing highly homologous proteins</li>
</ul>
</div>
</div>
<div id="postdoctoral-fellow" class="section level3">
<h3>Postdoctoral Fellow</h3>
<p>University of British Columbia</p>
<p>Vancouver, BC, Canada</p>
<p>2012 - 2009</p>
<div class="concise">
<ul>
<li>Studied the ubiquitin &amp; proteasome system</li>
<li>Mammalian cell culture</li>
<li>Immunofluorescence</li>
<li>Site-directed mutagenesis</li>
<li>Mentored several undergraduate researchers</li>
</ul>
</div>
</div>
</div>
<div id="education" class="section level2" data-icon="graduation-cap" data-concise="true">
<h2>Education</h2>
<div id="university-of-delaware" class="section level3">
<h3>University of Delaware</h3>
<p>Ph.D. in Biochemistry</p>
<p>Newark, DE, USA</p>
<p>2009 - 2002</p>
<p>Thesis advisor: Dr. Brian Bahnson. Research involved structural biology of G protein-coupled
receptors and Phospholipase A2 membrane-associated enzymes. Awarded an NSF Integrative
Graduate Research Traineeship (IGERT).</p>
</div>
<div id="florida-state-university" class="section level3">
<h3>Florida State University</h3>
<p>B.Sc. in Chemistry</p>
<p>Tallahassee, FL, USA</p>
<p>2002 - 1998</p>
</div>
</div>
<div id="selected-publications" class="section level2" data-icon="file">
<h2>Selected Publications</h2>
<div id="proteomic-and-bioinformatic-analyses-for-the-identification-of-proteins-with-low-allergenic-potential-for-hazard-assessment" class="section level3">
<h3>Proteomic and bioinformatic analyses for the identification of proteins with low allergenic potential for hazard assessment</h3>
<p>Toxicological Sciences 170 (1), 210-222</p>
<p>N/A</p>
<p>2019</p>
<p>Krutz NL, <strong>Winget JM</strong>, Ryan CA, Wimalasena R, Maurer-Stroh S, Dearman RJ, Kimber I, Gerberick GF</p>
</div>
<div id="stpeter-seamless-label-free-quantification-with-the-trans-proteomic-pipeline" class="section level3">
<h3>StPeter: Seamless label-free quantification with the trans-proteomic pipeline</h3>
<p>Journal of proteome research 17 (3), 1314-1320</p>
<p>N/A</p>
<p>2018</p>
<p>Hoopmann MR, <strong>Winget JM</strong>, Mendoza L, Moritz RL</p>
</div>
<div id="quantitative-proteomic-analysis-of-stratum-corneum-dysfunction-in-adult-chronic-atopic-dermatitis" class="section level3">
<h3>Quantitative proteomic analysis of stratum corneum dysfunction in adult chronic atopic dermatitis</h3>
<p>The Journal of investigative dermatology 136 (8), 1732</p>
<p>N/A</p>
<p>2016</p>
<p><strong>Winget JM</strong>, Finlay D, Mills KJ, Huggins T, Bascom C, Isfort RJ, Moritz RL</p>
</div>
<div id="quantitative-proteogenomic-profiling-of-epidermal-barrier-formation-in-vitro" class="section level3">
<h3>Quantitative proteogenomic profiling of epidermal barrier formation in vitro</h3>
<p>Journal of dermatological science 78 (3), 173-180</p>
<p>N/A</p>
<p>2015</p>
<p><strong>Winget JM</strong>, Watts JD, Hoopmann MR, DiColandrea T, Robinson MK, Huggins T, Bascom C, Isfort RJ, Moritz RL</p>
</div>
<div id="the-diversity-of-ubiquitin-recognition-hot-spots-and-varied-specificity" class="section level3">
<h3>The diversity of ubiquitin recognition: hot spots and varied specificity</h3>
<p>Molecular cell 38 (5), 627-635</p>
<p>N/A</p>
<p>2010</p>
<p><strong>Winget JM</strong> and Mayor T</p>
</div>
</div>
</div>

<script>
(function() {
  var i, s, j, el, els;
  var create_el = function(type, className, content, isHTML) {
    var el = document.createElement(type);
    if (className) el.className = className;
    if (content) {
      if (isHTML) {el.innerHTML = content;} else {el.innerText = content;}
    }
    return el;
  }

  // replace h2 title with h1
  var title = document.getElementById('title').querySelector('h2');
  title.parentNode.replaceChild(create_el('h1', false, title.innerHTML, true), title);

  // add the class .aside to #aside
  el = document.getElementById('aside');
  if (el) el.className += ' aside';

  // tweak class names of sections, and add icons
  els = document.getElementById('main').querySelectorAll('.level2');
  for (i = 0; i < els.length; i++) {
    el = els[i];
    if (i === 0 && el.id === 'title') continue;
    el.className += ' main-block';
    if (el.dataset['concise']) el.className += ' concise';
    s = el.dataset['icon'] || 'bookmark';
    el.firstElementChild.insertBefore(create_el('i', 'fa fa-' + s), el.firstElementChild.firstChild);
  }

  // tweak class names of blocks in sections
  els = document.getElementById('main').querySelectorAll('.level3');
  for (i = 0; i < els.length; i++) {
    el = els[i];
    if (el.parentNode.id === 'title') continue;
    el.className += ' blocks';
    el.innerHTML = '<div class="date"></div>' + '<div class="decorator"></div>' +
      '<div class="details"><header></header><div></div></div>' + el.innerHTML;
    var ps = el.querySelectorAll('p');
    // move the date paragraph to the date div
    if (ps.length >= 3) {
      s = ps[2].innerText;
      s = s === 'N/A' ? [] : s.split(' - ');
      el.removeChild(ps[2]);
      for (j = 0; j < s.length && j < 2; j++) {
        el.children[0].appendChild(create_el('span', false, s[j]))
      }
    }
    // move title, description, location to the details div
    (function(h) {
      h.appendChild(el.children[3]);
      var p = el.children[3];  // description
      if (p.innerText !== 'N/A') {
        h.appendChild(create_el('span', 'place', p.innerHTML, true));
      }
      el.removeChild(p);
      p = el.children[3];  // location
      if (p.innerText !== 'N/A') {
        s = create_el('span', 'location', p.innerHTML, true);
        s.innerHTML = '<i class="fa fa-map-marker-alt"></i> ' + s.innerHTML;
        h.appendChild(s);
      }
      el.removeChild(p);
    })(el.children[2].firstElementChild);
    // move the rest of content in a block into the last div of the details div
    s = el.children[2];
    while (j = s.nextSibling) {
      if (j.className === 'aside') { s = j; continue; }
      el.children[2].lastElementChild.append(j);
    }
  }
})();
</script>


  </body>
</html>
